Python/pywarpx/Bucket.py

```diff
@@ -59,7 +59,6 @@ def attrlist(self):
         for attr, value in self.argvattrs.items():
             if value is None:
                 continue
-            # --- repr is applied to value so that for floats, all of the digits are included.
             # --- The strip of "'" is then needed when value is a string.
```

Comment:
One more thing - this comment can be deleted.

Reply:
The comment on `'` stripping?
```suggestion
```